### PR TITLE
Pancake Sorting

### PR DESCRIPTION
**Pull request by Ajay0511**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Pancake Sort

### Description

Pancake sorting is the colloquial term for the mathematical problem of sorting a disordered stack of pancakes in order of size when a spatula can be inserted at any point in the stack and used to flip all pancakes above it

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Review by vatsalkesarwani12 — APPROVED** [review]
update readme file

**Review by SKAUL05 — APPROVED** [review]

File: Python/sort/pancakesorting.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            #first moving maximum element to beginning
+            flip(arr,i)
+
+            #now moving maximum element to end
+            flip(arr,length-1)
+        length-=1
+
+
+#Testing our function on a user array
+user_input = input("Enter input numers separated by a comma\n").strip()
```
Comment (siddharth25pandey) [inline Python/sort/pancakesorting.py]: Remove grammatical mistake

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

**Comment by Ajay0511** [comment]
@siddharth25pandey Sir i have corrected the grammatical errors ,please review .

**Review by siddharth25pandey — APPROVED** [review]
@Ajay0511 Good to go
Able to merge.
Thanks!

**Comment by siddharth25pandey** [comment]
@all-contributors please add @Ajay0511 for his code.

**Pull request merged**